HandsOnTutorial/1-hello-world.fsx

```diff
@@ -67,26 +67,4 @@ In case you run into trouble, this can be used to clear all process
 records in the cluster: 
 *)
 
-cluster.ClearAllProcesses()
-
-(**
-If you need to get really heavy, you can reset the cluster, which clears 
-all process state in queues and storage. Other storage is left unchanged.
-Your worker roles may need to be manually rebooted (e.g. from the Azure 
-management console).
-*)
-// cluster.Reset()
```

Comment:
The disappearance of cluster.Reset() is an omission that will be fixed. Logs can now be fetched on a per-task basis using `CloudTask.GetLogs`. System logs specific to the current process can be fetched by subscribing loggers to the cluster object using `cluster.AttachLogger`.